haskell/src/Stablecoin/Client/Main.hs

```diff
@@ -44,13 +45,13 @@ mainProgram :: ClientArgs -> MorleyClientM ()
 mainProgram (ClientArgs _ globalOptions cmd) = do
   case cmd of
     CmdDeployContract DeployContractOptions {..} -> do
-      let contractAlias = AliasHint "stablecoin"
-      -- TODO: we're using coerce in a few places to convert `AliasHint` to `Alias`
-      -- where necessary, but we shouldn't be doing this because it assumes no alias
-      -- prefix option was passed in via the CLI.
-      aliasAlreadyExists <- checkIfAliasExists (coerce contractAlias)
+      prefix <- tceAliasPrefix <$> view tezosClientEnvL
```

Comment:
a bit sad that this is tied to the logic inside morley client itself, but I suppose that there is little we can do if we want to print the full alias in the messages to the user.

There is some space to remove duplication in morley client itself tho, I'll open an issue for this there.

Reply:
> a bit sad that this is tied to the logic inside morley client itself

Maybe it's worth adding a convenient way to get full alias with prefix to `morley-client` ? :thinking: 

Btw, I noticed a piece of code which moved to a separate function in `morley-client` (namely `setupMoneybagAddress`), and we can use it there when we update `morley` version. Should I left a `TODO` for this purpose, or it's better to add a comment to issue related to bumping `morley` dependency (but I can't find it :sweat_smile:)